src/cdk/overlay/position/global-position-strategy.ts

```diff
@@ -189,8 +193,23 @@ export class GlobalPositionStrategy implements PositionStrategy {
   }
 
   /**
-   * Noop implemented as a part of the PositionStrategy interface.
+   * Cleans up the DOM changes from the position strategy.
    * @docs-private
    */
-  dispose(): void { }
+  dispose(): void {
+    if (this._isDisposed) {
+      return;
+    }
+
+    const styles = this._overlayRef.overlayElement.style;
```

Comment:
Somehow there's a failing GCP test where `this._overlayRef` is undefined here

Reply:
I suppose that could happen if `dispose` somehow got called before `attach`. I've added an extra check in there.